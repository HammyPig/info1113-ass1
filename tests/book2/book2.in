ADD COLLECTION books.csv
book 999999
bOok 999999 lONG
BOOK 111111
bOOK 111111 LONG
book 111112 long
book 111112