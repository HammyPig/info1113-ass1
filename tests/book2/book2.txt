outputs short/long string, or if book is not found