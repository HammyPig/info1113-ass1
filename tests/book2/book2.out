user: 17 books successfully added.

user: No such book in system.

user: No such book in system.

user: To Kill a Mockingbird (Harper Lee)

user: 111111: To Kill a Mockingbird (Harper Lee, Historical Fiction)
Currently available.

user: 111112: The Hobbit (J.R.R. Tolkien, Fantasy)
Currently available.

user: The Hobbit (J.R.R. Tolkien)

user: Ending Library process.
